resource-managers/kubernetes/core/src/test/scala/org/apache/spark/deploy/k8s/submit/KubernetesClientUtilsSuite.scala

```diff
@@ -76,4 +80,26 @@ class KubernetesClientUtilsSuite extends SparkFunSuite with BeforeAndAfter {
       "testConf.3" -> "test123456")
     assert(output === expectedOutput)
   }
+
+  test("verify that configmap built as expected") {
+    val configMapName = "testConfigMapName"
+    val properties = Map(Config.KUBERNETES_NAMESPACE.key -> "default")
```

Comment:
If you don't mind, could you use a random name here? Although the previous code doesn't attache the namespace, this `default` is too generic to be in the test case.

Reply:
OK, I change it.